lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -466,6 +478,19 @@ def _container_security_context_is_set(
         return True
 
 
+class NadConfigChangedEvent(EventBase):
```

Comment:
Adding the custom charm event definition here to prevent to define it again and again in the charms.

Reply:
Removed from library